DatadogLogs/Sources/Feature/MessageReceivers.swift

```diff
@@ -191,6 +191,7 @@ internal struct CrashLogReceiver: FeatureMessageReceiver {
 
     /// Time provider.
     let dateProvider: DateProvider
+    let logEventMapper: LogEventMapper?
```

Comment:
/siggestion For convenience, we can have an `init` that default the mapper to `nil`

Reply:
I also prefer being explicit. This way compiler will guide us on all the places where `MessageReceiver` is used and will make us think of injecting mapper where it makes sense. If we seek for testing convenience, it is safer to add it through extension in test module.